dataprofiler/data_readers/data_utils.py

```diff
@@ -315,7 +315,7 @@ def reservoir(file: TextIOWrapper, sample_nrows: int) -> list:
     rng = random.Random(x=settings._seed)
     if "DATAPROFILER_SEED" in os.environ and settings._seed is None:
         seed = os.environ.get("DATAPROFILER_SEED")
-        if seed:
+        if isinstance(seed, int):
```

Comment:
matching how we check seed elsewhere

Reply:
Need to rethink. Good to be consistent, but I think the intent is to error if the seed isn't valid.

Current would silently not set a seed as opposed to error.

Reply:
should make an issue but revert

Reply:
@micdavis just bumping this.

Reply:
`os.environ.get("DATAPROFILER_SEED")` comes in as a str to begin with

Reply:
this still needs to be reverted.